infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/query/parsing/impl/InfinispanRemoteQueryBuilder.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.infinispanremote.query.parsing.impl;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.hibernate.HibernateException;
+
+/**
+ * Helper class to append single or multiple values to the wrapped {@link StringBuilder}
+ *
+ * @author Fabio Massimo Ercoli
+ */
+public class InfinispanRemoteQueryBuilder implements Serializable {
+
+	private StringBuilder builder;
+	private boolean where;
+
+	public InfinispanRemoteQueryBuilder() {
+		this.builder = new StringBuilder();
+	}
+
+	public InfinispanRemoteQueryBuilder(String content) {
+		this.builder = new StringBuilder( content );
+	}
+
+	public InfinispanRemoteQueryBuilder(String content, String otherContent) {
+		this.builder = new StringBuilder( content );
+		append( otherContent );
+	}
+
+	public InfinispanRemoteQueryBuilder(String content, InfinispanRemoteQueryBuilder child) {
+		this.builder = new StringBuilder( content );
+		append( child );
+	}
+
+	public void append(String content) {
+		builder.append( content );
+	}
+
+	public void append(InfinispanRemoteQueryBuilder child) {
+		builder.append( child.builder );
+	}
+
+	public void appendValue(Object value) {
+		boolean isConstant = value instanceof String;
+
+		if ( isConstant ) {
+			builder.append( "'" );
```

Comment:
Don't you need to check if the string contains `'` and escape them?

Reply:
I'm not sure about the `isCostant` can you give me an example? I would expect something like isText instead. Maybe I'm missing something?